data_model/ordinary_data/frame.sql

```diff
@@ -5,14 +5,14 @@
 CREATE TABLE siro_od.frame
 (
     id uuid PRIMARY KEY default uuid_generate_v1(),
+    rank int,
```

Comment:
```suggestion
    rank int unique default max(unique)+1,
```

je suis pas sûr de la syntaxe, mais il faut que la valeur soit unique et on peut essayer d'avoir une valeur par défaut